bpf/lib/nat.h

```diff
@@ -506,6 +506,9 @@ snat_v4_nat_can_skip(const struct ipv4_nat_target *target, const struct ipv4_ct_
 {
 	__u16 sport = bpf_ntohs(tuple->sport);
 
+	if (target->egress_gateway)
+		return false;
+
```

Comment:
can we wrap this in `ENABLE_EGRESS_GATEWAY` ?

Reply:
Whoops, fixed